### PR TITLE
#1241 June Routine upgrades

### PR DESCRIPTION
**Pull request by justaskdavidb2**
# Description

Added latest updates for python modules that will not cause failed tests

#1241

## Type of change

Please delete options that are not relevant.

-  New feature (non-breaking change which adds functionality) --> upgrades only

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Local Test Cases pass
- [x] All Unit Test Cases pass
- [x] Made sure DataDog received data from AWS
- [x] Twilio can send messages
- [x] Cloudwatch logs for errors

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -87,25 +85,24 @@ python-json-logger==2.0.7
 python-magic==0.4.27
 pytz==2023.3
 PyYAML==5.4.1
-redis==4.5.4
-requests==2.28.2
+redis==4.5.5
+requests==2.31.0
```
Comment (cris-oddball) [inline requirements.txt:89]: I thought we couldn't upgrade requests due to the alpine container version? Was this code deployed to Dev?

File: requirements.txt [hunk 2]
```diff
@@ -87,25 +85,24 @@ python-json-logger==2.0.7
...
+SQLAlchemy==1.4.48
 statsd==4.0.1
-tenacity==8.2.2
-twilio==8.0.0
-typing_extensions==4.5.0
+twilio==8.3.0
+typing_extensions==4.6.3
 Unidecode==1.3.6
-urllib3==1.26.15
+urllib3==1.26.16
```
Comment (cris-oddball) [inline requirements.txt:100]: I thought we also couldn't upgrade urllib3 due to the alpine container version - was this code deployed to dev?

**Review by cris-oddball — CHANGES_REQUESTED** [review]

**Review by kalbfled — CHANGES_REQUESTED** [review]
Do these changes clear any vulnerabilities identified by Safety?  I'm pretty sure they do.  I would expect additional changes in Makefile to remove currently ignored vulnerabilities.

Please also note that this branch needs to be rebased to master.  I think that's why the CodeQL check says, "configuration not found."  That workflow is running from master, but your branch doesn't have the relevant files.

File: requirements-app.txt [hunk 3]
```diff
@@ -25,13 +25,17 @@ fido2 >= 1.1.1
 Flask >= 2.2.5
 Flask-Bcrypt >= 1.0.1
 Flask-Cors >= 3.0.10
+
+# 2023 June 21: Created issue ticket for upgrading Flask-JWT-Extended
+# Cannot upgrade because of breaking changes causing "Attribute Error: 'Cookie' has no attribute 'name'"
```
[the remark this answers is not in the document]
Comment (cris-oddball) [reply requirements-app.txt:30]: With the update, you now have duplicate text on line 30, line 31. At this point, just delete the comments on lines 29 and 30, the comments on 31 and 32 suffice.

**Review by cris-oddball — CHANGES_REQUESTED** [review]

**Review by cris-oddball — APPROVED** [review]

**Review by kalbfled — APPROVED** [review]

**Pull request merged**